java-dz/src/main/java/org/reckeles/db/UserData.java

```
@@ -0,0 +1,118 @@
+        String sql = "INSERT INTO " +
+                this.TABLE +
+                " (" + FIRST_NAME_COLUMN + ", " +
+                LAST_NAME_COLUMN + ", " +
+                LOGIN_COLUMN + ", " +
+                PASSWORD_HASH_COLUMN + ", " +
+                DATE_OF_BIRTH_COLUMN + ") " +
+                "VALUES (?, ?, ?, ?, STR_TO_DATE(?, '%d-%m-%Y'));";
+        long id = 0;
+        try {
+            PreparedStatement statement = Session.get().mysql().preparedStatement(sql);
+            statement.setString(1, firstName);
+            statement.setString(2, lastName);
+            statement.setString(3, login);
+            statement.setString(4, passwordHash);
+            statement.setString(5, birthDate);
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            String msg = "Could not update first name ";
+            logger.error(msg + ": REASON: " + e.getMessage());
+        }
+    }
+
+    public User getUser(int id) {
+        String sql = "SELECT * FROM " + this.TABLE + " WHERE id=?;";
+        try {
+            PreparedStatement statement = Session.get().mysql().preparedStatement(sql);
+            statement.setString(1, String.valueOf(id));
+            ResultSet result = statement.executeQuery();
+            if (!result.next()) {
+                return null;
+            }
+
+            return new User(
+                    result.getInt(ID_COLUMN),
+                    result.getString(FIRST_NAME_COLUMN),
+                    result.getString(LAST_NAME_COLUMN),
+                    result.getString(LOGIN_COLUMN),
+                    result.getString(PASSWORD_HASH_COLUMN),
+                    result.getDate(DATE_OF_BIRTH_COLUMN).toLocalDate()
+            );
+        } catch (SQLException e) {
+            String msg = "Could not get user data ";
+            logger.error(msg + ": REASON: " + e.getMessage());
+            return null;
```

Better to throw `RuntimeException` instead of returning null, because in this case you can get `NullPointerException` on run and need to find time of execution and find the reason in logs.